mithril-aggregator/src/dependency_injection/builder.rs

```diff
@@ -746,7 +746,8 @@ impl DependenciesBuilder {
     }
 
     async fn build_transaction_parser(&mut self) -> Result<Arc<dyn TransactionParser>> {
-        let transaction_parser = CardanoTransactionParser::new(self.get_logger().await?);
+        // TODO: 'allow_unparsable_block' parameter should be configurable and its default value set to false
+        let transaction_parser = CardanoTransactionParser::new(self.get_logger().await?, true);
```

Comment:
This would be more readable:
```suggestion
        let allow_unparsable_block = true
        let transaction_parser = CardanoTransactionParser::new(self.get_logger().await?, allow_unparsable_block);
```